.github/workflows/static-code-analysis.yml

```diff
@@ -7,10 +7,6 @@ on:
     branches:
       - dev
 
-defaults:
-  run:
-    shell: pwsh
-
 jobs:
   powershell-static-code-analysis:
```

Comment:
The whole job, also in Infrastructure Scripts and PowerShell Analyzers if necessary, could be a reusable workflow in PowerShell Analyzers.

Reply:
I wanted to suggest the same (#showerthoughts)! Would also make the DX better, because PS Analyzers is a submodule of both OSOCE and Utility Scripts, which makes it a bit difficult to work with. On a related note, Utility Scripts shouldn't be a submodule of Infrastructure Scripts anymore either (it was just a convenient shortcut for me when it was still closed source).